test_infra/stacks/databases_stack.py

```diff
@@ -493,7 +493,7 @@ def _setup_mysql(self) -> None:
                 "USERNAME": self.db_username,
                 "PASSWORD": self.db_password,
                 "JDBC_ENFORCE_SSL": "true",
-                "CUSTOM_JDBC_CERT": "s3://rds-downloads/rds-combined-ca-bundle.pem",
+                "CUSTOM_JDBC_CERT": "s3://aws-glue-assets-658066294590-us-east-1/certificates/global-bundle.pem",
```

Comment:
Hmm, that would would break if we migrate our testing to a new account or if a customer try to run the tests

Reply:
Unfortunately there isn't. I've asked RDS team if the bucket is still managed and if they would add the new certs there, waiting for the reply, but meanwhile as a workaround looks like we will have to host it somewhere on S3 ourselves.

Reply:
Maybe we could create a separate bucket where we store the certificate and then make this object publicly readable? That way, if a customer deploys the test infra to their own account, they'll still be able to connect to their DBs thanks to the certificate we're providing.

Reply:
We already have an existing public bucket so might as well use that one. It's complicated to create public buckets these days

Reply:
There is also `CUSTOM_JDBC_CERT_STRING` as an alternative which I think is base64-encoded .PEM, I could try bake that in instead.

Reply:
Getting `[SSL: CERTIFICATE_VERIFY_FAILED]` with `CUSTOM_JDBC_CERT_STRING ` - I've reverted to use the cert in the bucket.